pages/portfolio.html

```diff
@@ -1,12 +1,227 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link href="/css/portfolio.css" rel="stylesheet" />
```

Comment:
Careful with your file paths! The `/styles.css/style.css` path does not actually work because it is an incorrect absolute path. Starting a path with a slash (/) will start the file search at the root directory of your entire file system (not your project).
To correct this, we can use a relative path instead. This will provide directions to the target file based on the current file's location.
In code this looks like: `../styles.css/style.css`. The two dots at the beginning signal to start searching in the parent directory of the current directory.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/

This is true of links to other pages on the website and of links to images.

Reply:
Hi Nancy! Thanks so much for reviewing my code and for the resource on relative paths! I couldn't find the /styles.css/style.css in my html files, so I just wanted to confirm this this feedback was for me or if it was just a typo?
I found this link to a CSS stylesheet though /css/portfolio.css, should it have been "../css/portfolio.css" instead?

Reply:
oops sorry! I found the bug in the index.html file, it looks like I added a slash in front of "css/style.css" and now it's working. Thanks so much for flagging this for me :)

Reply:
Hi Milena! Sorry, I copy-pasted that comment from somewhere else and didn't update it to match what you have in your code. But yes, for your index.html file, that change will fix it! It'll be a bit different for the other pages though, because those are embedded in the pages folder. In order to get out of the pages folder and into the css one, you would need the two dots at the beginning. So `../css/portfolio.css` would be the correct thing to do if you're in the portfolio.html file.

Reply:
Oh no worries!! Understanding paths better now has been helping me with my capstone, so thanks again for pointing that out and helping me fix it :)